AmplifyPlugins/Auth/Sources/AWSCognitoAuthPlugin/StateMachine/CodeGen/Events/SignInEvent.swift

```diff
@@ -72,6 +74,7 @@ struct SignInEvent: StateMachineEvent {
         case .throwAuthError: return "SignInEvent.throwAuthError"
         case .receivedChallenge: return "SignInEvent.receivedChallenge"
         case .verifySMSChallenge: return "SignInEvent.verifySMSChallenge"
+        case . retryRespondPasswordVerifier: return "SignInEvent.retryRespondPasswordVerifier"
```

Comment:
Does this even compile?

```suggestion
        case .retryRespondPasswordVerifier: return "SignInEvent.retryRespondPasswordVerifier"
```

Reply:
It does compile ¯\_(ツ)_/¯